src/consumer/component/dependencies/dependency-resolver/dependencies-resolver.ts

```diff
@@ -764,6 +771,10 @@ either, use the ignore file syntax or change the require statement to have a mod
     });
   }
 
+  private isPkgInWorkspacePolicies(pkgName: string) {
+    return DependencyResolver.getWorkspacePolicy().dependencies?.[pkgName];
```

Comment:
What about dev/peer deps? maybe we need to check whether it's in one of them?

Reply:
There is no separate field for devDependencies. Regarding peerDependencies. Good question. I assume it is fine to use the dependency from the model in case of peer deps because peer deps allow a wide range of dependencies.